src/page/MoviePage.jsx

```diff
@@ -1,16 +1,30 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
+import { connect } from 'react-redux';
+
 import { Paper } from '@material-ui/core';
 
-import { PAGE_COUNT } from '../constants/constants';
-import { MOVIES } from '../constants/actions';
-import Search from '../containers/Search';
-import MovieList from '../components/MovieList/MovieList';
+import { PAGE_COUNT, PARAMS as params } from '../constants/constants';
+import Search from '../components/Search';
+import MovieList from '../components/MovieList';
+import { movies } from '../actions';
 
-import { getMovies, getSearchMovies } from '../api/api';
+import { getMovies } from '../api/api';
 import { FlatPagination } from '../components/FlatPagination/FlatPagination';
 
+@connect(
+  ({ search, moviePage }) => ({
+    ...moviePage,
+    ...search,
+  }),
+  dispatch => ({
```

Comment:
too complex code. Use object instead like it's done here
https://react-redux.js.org/using-react-redux/connect-mapdispatch 
(read paragraph **Defining mapDispatchToProps As An Object**)